e2e/test/scenarios/joins/reproductions/14793-xrays-joins.cy.spec.js

```diff
@@ -66,7 +66,7 @@ describe("issue 14793", () => {
 
     dashboardGrid()
       .findByText("How this metric is distributed across different numbers")
-      .should("be.visible");
+      .should("exist");
 
```

Comment:
The addition of new cards from metrics can make the desired card move further down on the page and be out of the test harness viewport. It is perfectly legitimate for the card to be present but not visible.